### DISCUSSION TITLE
user-guide/citation-styles

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#83] [文档评论区]
# user-guide/citation-styles

Zotero 使用 Citation Style Language 来生成引文格式，Zotero 官方提供了一组 CSL 格式，可以在 Zotero 内下载安装。但是官方的引用格式对国内期刊支持不全，且目前（CSL 1.0.2）仍未支持双语排版（“et al”与“等”混排），所以国内有热心开发者维护了一份第三方引文格式，读者可以按需下载使用。 官方引文...

https://zotero-chinese.com/user-guide/citation-styles.html

**Comment by 18688450783** [comment]
我这样添加了引文格式之后，在引文处可以出现上角标，但是文档下面没有参考文献列表，这是怎么回事？我要怎么操作呢？

**Reply by northword** [reply]
下一页，在word中添加参考文献，找 添加参考文献表 这一小节
https://zotero-chinese.com/user-guide/ms-word-plugin.html#%E6%B7%BB%E5%8A%A0%E5%8F%82%E8%80%83%E6%96%87%E7%8C%AE%E8%A1%A8

**Comment by Eva-8866** [comment]
从Github下载了，但是zotero中没有查找到，又使用右击另存为，但是从引文添加显示安装时发生未可预知的错误，想问下怎么解决

**Comment by LitchiRuirui** [comment]
你好，想请问一下，安装了人民出版社的csl样式后，为什么有的期刊条目可以显示正确效果，有的就不行呢？

**Reply by northword** [reply]
https://github.com/zotero-chinese/styles/issues/402

**Comment by xuweiii98** [comment]
也没有对中文文献自动生成英文对照，满足中英文混排的GB2015格式

**Comment by windbd** [comment]
⭐⭐⭐在这个网站https://editor.citationstyles.org/about/ 中开可以自己设计CLS引文格式并进行下载

**Comment by crt2441797495** [comment]
你好，在使用最新的zotero7之后，原本支持双语的csl现在都不行了，我打开样式编辑器之后显示attribute ^locale not allowed，这要怎么解决啊

**Reply by crt2441797495** [reply]
不好意思，搞错了，换版本之后文件导入有点问题，重新安装就好了

**Reply by Clouds42** [reply]
我也有一样的问题，请问你最终是如何解决的？我重新安装Zotero7仍然会报错，并且在Word中插入文献后会闪退。

**Comment by crt2441797495** [comment]
我之前还重新安装了一下word插件，然后论文的语言那边英文的都改成en就行了

**Comment by 111ruirui** [comment]
按照“2.2 先下载文件后安装”的方法安装csl文件后，可以正常使用，但是重启Zotero后，原先安装的csl文件就不见了，还得重新安装，请问怎么解决呀

**Comment by wxybb** [comment]
请问如何切换文中引文格式？如将wang(2010)切换成(wang,2010)

**Reply by zepinglee** [reply]
使用的哪个样式？大部分默认是 (wang,2010) 的。

**Comment by yasi-hub** [comment]
GB/T 7714--2015(顺序编码，双语，姓名不大写)——官网这个版本的总带着OL-[J/OL].可以整一版不带OL的吗？如下面的格式，谢谢

[1] 窦家莹,周益平,崔云,等.儿童重症监护病房血流感染严重脓毒症的病原学特点及死亡的影响因素分析[J].中华医学杂志,2024,104(3):198-204.DOI:10. 3760/cma.j.cn112137-20230729-00115.
[2] Essajee F,Solomons R,Goussard P,et al.Child with tuberculous meningitis and COVID-19 coinfection complicated by extensive cerebral sinus venous thrombosis[J].BMJ Case Rep,2020,13(9):e238597.DOI:10.1136/bcr-2020-2385 97.

**Reply by zepinglee** [reply]
> GB/T 7714--2015(顺序编码，双语，姓名不大写)——官网这个版本的总带着OL-[J/OL].可以整一版不带OL的吗？如下面的格式，谢谢
> 
> [1] 窦家莹,周益平,崔云,等.儿童重症监护病房血流感染严重脓毒症的病原学特点及死亡的影响因素分析[J].中华医学杂志,2024,104(3):198-204.DOI:10. 3760/cma.j.cn112137-20230729-00115. [2] Essajee F,Solomons R,Goussard P,et al.Child with tuberculous meningitis and COVID-19 coinfection complicated by extensive cerebral sinus venous thrombosis[J].BMJ Case Rep,2020,13(9):e238597.DOI:10.1136/bcr-2020-2385 97.

https://zotero-chinese.com/styles/gb-t-7714-2015-numeric-bilingual-no-uppercase-no-url-doi/

国标的示例中，有 DOI 的都是有「/OL」的。

**Reply by yasi-hub** [reply]
但是我们学校毕业论文的参考文献要求是这样的，有DOI没有[/OL]，可以求下面这个样式吗？谢谢~
[1] 窦家莹,周益平,崔云,等.儿童重症监护病房血流感染严重脓毒症的病原学特点及死亡的影响因素分析[J].中华医学杂志,2024,104(3):198-204.DOI:10. 3760/cma.j.cn112137-20230729-00115.
[2] Essajee F,Solomons R,Goussard P,et al.Child with tuberculous meningitis and COVID-19 coinfection complicated by extensive cerebral sinus venous thrombosis[J].BMJ Case Rep,2020,13(9):e238597.DOI:10.1136/bcr-2020-2385 97.

**Reply by zepinglee** [reply]
> 但是我们学校毕业论文的参考文献要求是这样的，有DOI没有[/OL]，可以求下面这个样式吗？谢谢~

不妨在 <https://github.com/zotero-chinese/styles/issues> 按照新样式发 issue。

**Reply by yasi-hub** [reply]
好的谢谢~

**Comment by oushiei120** [comment]


**Reply by zepinglee** [reply]
> 求《红楼梦》学刊的①②③格式的csl

<https://github.com/zotero-chinese/styles/issues> 按照格式发 issue。

**Comment by Shipshipss** [comment]
您好，我在引用窗格里（就是那个红色的框）显示的样式都是错误的，只跟随zotero的语言显示（比如中文就都显示等，英文都显示et al，无论文献是中文还是英文），但是虽然显示错误，导入到word里是正常的双语样式，请问您知道这个怎么解决吗？

**Comment by zhuyafei888** [comment]
如果一篇期刊，缺少卷号，那么该篇参考文献引用的格式，按照GB/T7714-2015来说，年和期号之间需要用逗号隔开，但是目前的参考文献格式都没分开，这是很大的bug。

**Reply by northword** [reply]
![Screenshot_2025-02-24-23-37-39-660_org.zotero.android-edit.jpg](https://github.com/user-attachments/assets/c47845e8-f479-417d-b741-77ce4be195fd)

有何问题？请给出具体示例，并参 https://zotero-chinese.com/user-guide/citation-styles#%E6%8A%A5%E5%91%8A%E6%A0%B7%E5%BC%8F%E9%94%99%E8%AF%AF 打开 issue 报告错误。

**Reply by zepinglee** [reply]
> 这是很大的bug。

麻烦先调查清楚。

> 如果一篇期刊，缺少卷号，那么该篇参考文献引用的格式，按照GB/T7714-2015来说，年和期号之间需要用逗号隔开，但是目前的参考文献格式都没分开，

我再补充 9.1 节的示例，以及标准起草人的文章 《[GB/T 7714新标准对旧标准的主要修改及实施要点提示](http://bjxb.cessp.org.cn/ch/reader/view_abstract.aspx?file_no=20150411&flag=1)》。

<img width="775" alt="Screenshot 2025-02-25 at 06 22 32" src="https://github.com/user-attachments/assets/6f44476e-38b5-4dea-9c14-d2cd90511dec" />

<img width="461" alt="Screenshot 2025-02-25 at 06 24 46" src="https://github.com/user-attachments/assets/ebcf51ee-5fe3-40ec-809e-636e50ef22c9" />

**Comment by alexzorro561** [comment]
感谢！

**Comment by Hhhls** [comment]
请求帮助：下载的
gb-t-7714-2015-numeric-bilingual-no-uppercase-no-et-al-no-initialization.csl
不想要DOI和URL，怎么修改啊？求助！

**Reply by zepinglee** [reply]
从 <https://zotero-chinese.com/styles/gb-t-7714-2015-numeric-bilingual-no-uppercase-no-url-doi/> 修改「全部作者，姓名不缩写」更容易些。

**Comment by ABad-Boy** [comment]
请问如何上传新的试样？因为有的期刊使用国标，但是要求使用期刊缩写，目前的所有国标版本均不可使用期刊缩写，所以我在国标的基础上修改了一个可以使用期刊缩写的试样，打算上传中文库

**Reply by northword** [reply]
样式全部在 zotero-chinese/styles 仓库管理。

打开 https://github.com/zotero-chinese/styles/tree/main/src ，点击 Add File -> Create new file:
<img width="1912" height="924" alt="PixPin_2025-10-21_22-37-57" src="https://github.com/user-attachments/assets/6a2bfdba-eb31-4ba5-9853-366544e6306d" />

这时候可能会提示没有权限需要 fork，点击确认（这里的流程与文档十分相近，可以参考 https://zotero-chinese.com/contributing/wiki#%E6%96%B9%E6%B3%95-1-%E5%9C%A8-github-%E4%B8%8A%E5%9C%A8%E7%BA%BF%E4%BF%AE%E6%94%B9 ）。

输入文件路径和 CSL 代码（记得改 info 里的 id、title、link 等信息）：

<img width="1905" height="607" alt="PixPin_2025-10-21_22-37-31" src="https://github.com/user-attachments/assets/217a91ba-0b41-4f99-8a05-58849c5a2678" />

右上角绿色按钮提交更改。根据网页引导发起 PR。

机器人会自动生成预览文件。

**Comment by shuaishuaichoose** [comment]
感谢，辛苦各位开发者

**Comment by iamAnn123** [comment]
为什么我使用了GB7714/2015（著者，出版年）样式，使用之后参考文献全按照ABCD排序了，中文的跑到最前面了，不是按照引用顺序呢

**Reply by zepinglee** [reply]
> 为什么我使用了GB7714/2015（著者，出版年）样式，使用之后参考文献全按照ABCD排序了，中文的跑到最前面了，不是按照引用顺序呢

<img width="809" height="322" alt="Screenshot 2025-11-08 at 18 06 04" src="https://github.com/user-attachments/assets/863c36fd-2591-413e-befb-bed9a6647eaa" />

如何按引用顺序，你打算让读者怎么按「著者-出版年」索引文献？

**Reply by iamAnn123** [reply]
我明白了，搞错了，谢谢。

**Comment by ABad-Boy** [comment]
解决了就好，实在不好意思，我刚看到信息


---- Replied Message ----
| From | ***@***.***> |
| Date | 11/8/2025 20:36 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [zotero-chinese/wiki] user-guide/citation-styles (Discussion #83) |

我明白了，搞错了，谢谢。

**Comment by 2026-lx** [comment]
我想问一下7714-2005引文上标是这样表示的【1,2】，但是期刊文献是【1-2】，是我选的引文格式不对吗，换成7714-2015或2025都是这样，我要怎么修改成【1-2】表示呢

**Reply by zepinglee** [reply]
按照国标确实是“1-2”，但是目前 CSL 无法实现，因为 [citeproc-js](https://github.com/Juris-M/citeproc-js) 的逻辑写死了：连续 3 个以上才会用连接号。

**Comment by ABad-Boy** [comment]
您好，这个我没有研究过，因为在我看来这两个是一样的，您可以看看他的源代码，可能会有所发现，我所有的修改也是在源码上进行的改动。祝好


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2026年01月08日 11:57 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [zotero-chinese/wiki] user-guide/citation-styles (Discussion #83) |

我想问一下7714-2005引文上标是这样表示的【1,2】，但是期刊文献是【1-2】，是我选的引文格式不对吗，换成7714-2015或2025都是这样，我要怎么修改成【1-2】表示呢

**Comment by ABad-Boy** [comment]
如果是代码逻辑写死那就没办法了，只能找一个和需求相似的文件进行修改，或者直接在csl修改器里重新创建一个


---- Replied Message ----
| From | Zeping ***@***.***> |
| Date | 1/10/2026 18:33 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [zotero-chinese/wiki] user-guide/citation-styles (Discussion #83) |

按照国标确实是“1-2”，但是目前 CSL 无法实现，因为 citeproc-js) 的逻辑写死了：连续 3 个以上才会用连接号。

**Comment by xuruixue11-crypto** [comment]
有没有中英对照的引文格式呢？

**Comment by ABad-Boy** [comment]
目前应该是没见过这种格式，只能自己创建一个


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2026年01月27日 23:04 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [zotero-chinese/wiki] user-guide/citation-styles (Discussion #83) |

有没有中英对照的引文格式呢？

**Comment by fusaer** [comment]
请教：
[1]	中华人民共和国交通运输部. 水运工程静力触探技术规程: JTS/T 242-2020[S]. 北京: 人民交通出版社, 2021.
怎么将标准编号的位置调整到标题前面？然后条目类型还是在标题后面？
就是这样的形式：
中华人民共和国交通运输部.  JTS/T 242-2020 水运工程静力触探技术规程[S]. 北京: 人民交通出版社, 2021.

**Reply by zepinglee** [reply]
使用 2025 版的样式。

2015 版的标准号在题名和冒号后。

**Comment by WjqGW2020** [comment]
GB-T-7714—2015（顺序编码，双语，无URL、DOI）,怎样才能按照“中文在前，英文在后”进行排序？

**Reply by zepinglee** [reply]
国标的著者-出版年制才需要「中文在前，英文在后」吧？顺序编码制是按照引用的先后顺序。

**Comment by dinnercooker** [comment]
建议样式进行整合，可以自动选择需求，形成样式，要不太乱。比如默认是什么样，然后根据需要你可以点后面添加，然后自动生成或者引导到早就生成的里去，现在符合GB-T-7714—2015缺少个全作者，且无doi链接那种的。

**Reply by zepinglee** [reply]
你的意思是做一个网页版的 GB/T 7714 CSL 样式生成器，允许用户配置选项？

**Reply by dinnercooker** [reply]
> 你的意思是做一个网页版的 GB/T 7714 CSL 样式生成器，允许用户配置选项？

是的，一劳永逸解决这个问题，目前太多样式了，而且不知道2015和2025的区别，其实核心就是要不要后面的DOI，和要不要前面完整作者。如果有个直接一键生成多好，都是固定的代码的。

**Comment by wenxueman** [comment]
有没有csl可以让著作、期刊、学位论文分开排列，并且按照姓氏首字母排列呢
